docs/site/Using-strong-error-handler.md

```diff
@@ -17,8 +17,9 @@ In LoopBack 4, the request handling process starts with the app's
 [sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
 five injected helper methods in the constructor. It is the gatekeeper of all
 requests to the app. Errors are handled by one of the Sequence actions,
-[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
-package to send back an HTTP response describing the error.
+[`reject`](REST-middleware-sequence.md#handling-errors), which calls
```

Comment:
I chose middleware doc here because it is what we recommend now (not the action one)